src/tracer/Tracee.h

```diff
@@ -5,6 +5,14 @@
 #include <unistd.h>
 #include <sys/user.h>
 
+#if defined(__x86_64__)
+using reg_t = uint64_t;
```

Comment:
Jeszcze, jak to pisałem, to sobie pomyślałem, że można by nazwać 'ureg_t' :D @Wolf480pl  jak sądzisz?

Reply:
Hm... rejestry same z siebie nie są ani signed ani unsigned, decyduje sposób w jaki się do nich odwołujesz, a tu się odwołujemy jako unsigned, czyli `ureg_t`... z drugiej strony `reg_t` wygląda ładnie... dunno really.

Reply:
Dobra, zostawie reg_t :D